src/js/view/controls/controlbar.js

```diff
@@ -209,6 +209,10 @@ export default class Controlbar {
             }
             captionsTip.setText(newText);
         };
+        const playPauseStopTooltip = SimpleTooltip(elements.play.element(), 'play', localization.play);
+        this.onPlaybuttonTooltipChange = (newLocalization) => {
```

Comment:
We usually start method names with `on` for event handlers. Since this is a method you call to update the text wouldn't `setText` or `updateText` be more appropriate? Keep in mind that properties and methods are not minified either so keep them brief. vars and args can be as verbose as you like.

Reply:
I want it to be self documenting so no one has to look it up in the future. I think a good compromise is "setPlayText", so I've updated it to that and pushed the changes.